src/scenes/Scene.js

```diff
@@ -2,13 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-THREE.Scene = function () {
+THREE.Scene = function (fog) {
```

Comment:
If you must add optional parameters to the constructor, maybe the API should be

    var scene = new THREE.Scene( { fog: fog } );